package.json

```diff
@@ -53,34 +71,53 @@
     "karma-tap-pretty-reporter": "4.2.0",
     "karma-webpack": "git://github.com/ryanclark/karma-webpack#eec6dfc",
     "kw-doc": "2.0.0",
-    "kw-web-suite": "11.0.0",
+    "moment": "2.29.1",
     "node-pre-gyp": "0.16.0",
+    "parallel-webpack": "2.6.0",
+    "postcss-loader": "4.1.0",
+    "prettier": "2.2.1",
     "process": "0.11.10",
+    "raw-loader": "4.0.2",
     "request": "2.88.2",
     "resemblejs": "3.2.5",
+    "semantic-release": "17.3.1",
+    "shader-loader": "1.3.1",
     "stream-browserify": "3.0.0",
+    "string-replace-loader": "3.0.1",
+    "style-loader": "2.0.0",
     "tap-markdown": "1.2.1",
     "tap-spec": "5.0.0",
     "tape": "5.0.1",
     "tape-catch": "1.0.6",
+    "terser-webpack-plugin": "5.0.3",
```

Comment:
is it still the way to go to minify es6+?

Reply:
Turns out webpack v5 comes with terser bundled by default, so I'll make that change.